ssl/s3_lib.c

```diff
@@ -838,8 +838,7 @@ static SSL_CIPHER ssl3_ciphers[] = {
      1,
      TLS1_3_TXT_AES_128_GCM_SHA256,
      TLS1_3_CK_AES_128_GCM_SHA256,
-     SSL_kRSA,
-     SSL_aRSA,
+     0, 0,
```

Comment:
Can't think of any reason. I can do that in a separate PR or you can if you wish.